### ISSUE TITLE
Question about the output of loss3

### ISSUE THREAD

**Issue by skyzzzzo** [zhangjun001/ICNet#7]
Hi, I noticed after running the train.py file that the output of loss3 (also known as loss_antifold) in the terminal is always 0. For this reason, I checked the loss.npy file and found that only the first few times the value is 0, the rest have values. 
You can refer to the image below for more details, I simply modified the value of iteration to make it equal to 100.
![Snipaste_2024-08-28_16-24-47](https://github.com/user-attachments/assets/b343bc7f-0f30-42a6-8c03-8a5a9089ea53)
![1](https://github.com/user-attachments/assets/9a02efb7-b105-4059-b30b-75b63b1f8166)

I also set the weights of the loss function according to the paper, along with iteration=1000 and trained, and the output loss3 is still 0.

Therefore, I am very curious as to why there are values in the loss.npy but the terminal outputs 0. I hope to get your answer. 

Thanks!
